diagnostic.md

```diff
@@ -7,36 +7,36 @@ What command do we use to do that? Use the **correct** URL from your fork of
 this repository in your answer.
 
 ```sh
-<!-- Remove this comment and place your answer here. -->
+<!-- git clone git@github.com:dvdpwll/git-basics-diagnostic.git -->
 ```
 
 2.What do you do after cloning a repository, before you start making any
 changes/additions?
 
-<!-- Remove this comment and place your answer here. -->
+<!-- npm install -->
```

Comment:
Looking for branch, but this is true so far, sometimes we'll use other commands to install